src/incubator/Slider/SliderPresenter.ts

```diff
@@ -58,7 +58,6 @@ export function validateValues(props: SliderProps) {
 export function getStepInterpolated(trackWidth: number, minimumValue: number, maximumValue: number, stepXValue: SharedValue<number>) {
   'worklet';
   const outputRange = [0, trackWidth];
-  const inputRange = minimumValue < 0 ? 
-    [Math.abs(maximumValue), Math.abs(minimumValue)] : [minimumValue, maximumValue];
+  const inputRange = minimumValue < 0 ? [0, Math.abs(minimumValue)] : [0, maximumValue];
```

Comment:
I see that it works, but Im not sure I understand the bug. 
Can you explain what happens here that caused this?

Reply:
Ok, I added refinement to explain: The interpolation finds a value in one range at a different range. In this case, the value is the `stepXValue`, the x point equal to the step. This value is used to calculate the new location from the current location - where to jump to. Since this is an x value it doesn't care if the initial value is 12, it needs to start from 0 to give a value of x, so instead of defining the range 12 to 50, I'll define it as 0 to 38.

Reply:
👍